package.json

```diff
@@ -1,13 +1,14 @@
 {
-  "name": "{{project_name}}-frontend",
+  "name": "theprojectname-frontend",
   "version": "0.0.1",
   "private": true,
-  "description": "{{project_name}} frontend.",
+  "description": "theprojectname frontend.",
```

Comment:
I believe you can't edit this variable `{{project_name}}` here, otherwise, it won't be overridden when people bootstrap the boilerplate.

Bootstrapping the boilerplate with a project's name `test` would make this become `test-frontend`, but now it'll keep just `theprojectname-frontend` I think. Is that correct?

Reply:
I left this by mistake. Nice catch :D

Reply:
Hahah, I saw this and thought it was some new way you guys had defined to do this name replacement, so I didn't comment on it 😂